apps/web/vibes/soul/form/checkbox-group/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { clsx } from 'clsx';
+import * as React from 'react';
+
+import { FieldError } from '@/vibes/soul/form/field-error';
+import { Label } from '@/vibes/soul/form/label';
+
+import { Checkbox } from '../checkbox';
+
+interface Option {
+  value: string;
+  label: string;
+  disabled?: boolean;
+}
+
+interface Props {
+  id?: string;
+  className?: string;
+  label?: string;
+  options: Option[];
+  errors?: string[];
+  name?: string;
+  value: string[];
+  onValueChange: (value: string[]) => void;
+}
+
+export function CheckboxGroup({
+  className,
+  label,
+  options,
+  errors,
+  name,
+  value,
+  onValueChange,
+}: Props) {
+  return (
+    <div className={clsx('space-y-2', className)}>
+      {label !== undefined && label !== '' && <Label>{label}</Label>}
```

Comment:
Missing `htmlFor`

Reply:
What is the label for in a group of inputs?

Reply:
Done.